src/transactions/feed/NftFeedItem.tsx

```diff
@@ -35,28 +34,26 @@ function NftFeedItem({ transaction }: Props) {
     <Touchable testID={'NftFeedItem'} disabled={false} onPress={openNftTransactionDetails}>
       <View style={styles.container}>
         {/* Try to show the first image. Otherwise display the default icons */}
-        <IconWithNetworkBadge networkId={transaction.networkId}>
```

Comment:
Is this change necessary? We aren't supporting NFTs in mento right so this code shouldn't be reached?

Reply:
yes, not necessary but just did it anyway to be complete

Reply:
Should it also be removed from `PositionIcon` and `cardDetails` for completeness, and then comment out / remove component?

Reply:
this PR is scoped to tx feed items